oneview/resource_logical_interconnect_group.go

```diff
@@ -274,6 +280,163 @@ func resourceLogicalInterconnectGroup() *schema.Resource {
 					},
 				},
 			},
+			"sflow_configuration": {
+				Type:     schema.TypeList,
```

Comment:
Is TypeList required for this attribute? Can you point me to the documentation for this attribute. I'm not able to find it.

Reply:
its in 1000 and above versions for LIG